charts/opentelemetry-ebpf-instrumentation/templates/_helpers.tpl

```diff
@@ -0,0 +1,135 @@
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "obi.name" -}}
```

Comment:
iv'e replaced `beyla` with `obi` in all helpers since `ebpf-instrument.xyz` was kind of long, dont know if thats confusing